projects/example-app/src/app/books/containers/collection-page.component.ts

```diff
@@ -34,11 +36,9 @@ import * as fromBooks from '@example-app/books/reducers';
   ],
 })
 export class CollectionPageComponent implements OnInit {
-  books$: Observable<Book[]>;
+  private readonly store = inject(Store);
 
-  constructor(private store: Store) {
-    this.books$ = store.select(fromBooks.selectBookCollection);
-  }
+  readonly books = this.store.selectSignal(fromBooks.selectBookCollection);
```

Comment:
If I'm not mistaken this is probably added to test the behavior.
Is this intended to be left like this, or should this be undone (in favor of a signal based example app in the future)?

Reply:
Yea, I thought it was left there so we had one in the example app using a Signal, but I'm good either way